.shellcheckrc

```diff
@@ -32,3 +32,15 @@ disable=SC3013
 # https://github.com/koalaman/shellcheck/wiki/SC3045
 # ... but dash supports it
 disable=SC3045
+
+# SC2086: Double quote to prevent globbing and word splitting.
+# https://github.com/koalaman/shellcheck/wiki/SC2086
+disable=SC2086
+
+# SC2317: Command appears to be unreachable. Check usage (or ignore if invoked indirectly).
+# https://github.com/koalaman/shellcheck/wiki/SC2317
+disable=SC2317
+
+# SC2004: $/${} is unnecessary on arithmetic variables.
+# https://github.com/koalaman/shellcheck/wiki/SC2004
+disable=SC2004
```

Comment:
This one (`SC2004 `) seems completely reasonable to me; why would you disable this?